### PR TITLE
feat: [FRONT-58] added empty error message props for textfield

### PR DESCRIPTION
**Pull request by umrzoq-toshkentov**
While on this case I need to make red border textfield on error time, but I do not want to return any message
image

### CODE REVIEW

**Comment by doniyor2109** [comment]
@umrzoq-toshkentov Can you make something like this?

https://github.com/superdispatch/web-ui/blob/0fcf8f6f3e87c2e153ecf93dc3e3fe64e9dd333c/packages/forms/src/text/FormikCurrencyField.tsx#L10

File: packages/forms/src/text/FormikTextField.tsx [hunk 1]
```diff
@@ -74,7 +76,7 @@ export const FormikTextField: ForwardRefExoticComponent<FormikTextFieldProps> =
           ref={ref}
           name={name}
           error={!!errorText || errorProp}
-          helperText={errorText || helperText}
+          helperText={!emptyErrorMessage && (errorText || helperText)}
```
[the remark this answers is not in the document]
Comment (normurod72) [reply packages/forms/src/text/FormikTextField.tsx:79]: @umrzoq-toshkentov I think we can also not render helperText content when null or empty string provided, what do you think?

[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/forms/src/text/FormikTextField.tsx:79]: @timurhamitov I think I will remove this errorProps. I will use `EMPTY_ERROR_MESSAGE`. thanks to @doniyor2109

**Review by normurod72 — APPROVED** [review]

File: packages/forms/src/text/FormikTextField.tsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import {
   ForwardRefExoticComponent,
   ReactNode,
 } from 'react';
+import { EMPTY_ERROR_MESSAGE } from './FormikCurrencyField';
```
[the remark this answers is not in the document]
Comment (umrzoq-toshkentov) [reply packages/forms/src/text/FormikTextField.tsx]: created constants file and exported there

File: packages/forms/src/__tests__/index.spec.ts [hunk 3]
```diff
@@ -3,7 +3,6 @@ import * as api from '..';
 it('exposes public api', () => {
   expect(api).toMatchInlineSnapshot(`
     Object {
-      "EMPTY_ERROR_MESSAGE": "EMPTY_ERROR_MESSAGE",
```
Comment (doniyor2109) [inline packages/forms/src/__tests__/index.spec.ts:6]: @umrzoq-toshkentov We should not remove this constant from public APIs. Can you re export it?

**Review by doniyor2109 — CHANGES_REQUESTED** [review]

File: packages/forms/src/__tests__/index.spec.ts (hunk 3 above)
Comment (doniyor2109) [reply packages/forms/src/__tests__/index.spec.ts:6]: Let's include constayfile in index.ts file

**Review by doniyor2109 — APPROVED** [review]

**Pull request merged**